data/database.py

```diff
@@ -1391,8 +1391,8 @@ class ExternalNotificationMethod(BaseModel):
 class RepositoryNotification(BaseModel):
     uuid = CharField(default=uuid_generator, index=True)
     repository = ForeignKeyField(Repository)
-    event = ForeignKeyField(ExternalNotificationEvent)
-    method = ForeignKeyField(ExternalNotificationMethod)
+    event = EnumField(ExternalNotificationEvent)
```

Comment:
Doesn't this require a migration?

Reply:
Nope. EnumField is just a wrapper around ForeignKeyField